docs/source/markdown/podman-rename.1.md

```diff
@@ -0,0 +1,37 @@
+% podman-rename(1)
+
+## NAME
+podman\-rename - Rename an existing container
+
+## SYNOPSIS
+**podman rename** *container* *newname*
+
+**podman container rename** *container* *newname*
+
+## DESCRIPTION
+Rename changes the name of an existing container.
+The old name will be freed, and will be available for use.
+This command can be run on containers that are running without difficulty.
```

Comment:
Not sure what "without difficulty" means.  Can we list the states that a container can be in instead?  Or perhaps if it's easier, which state(s) the container can NOT be in?